### PR TITLE
449 workspace ui permissions

### PR DESCRIPTION
**Pull request by gyst**
Tries to fix workspace UI permissions, undoing the fine grained permission checks and using only the workflow managed cmf.AddPortalContent.

Doesn't work though. May indicate a deeper security issue in the "Consume" policy.

### CODE REVIEW

**Comment by gyst** [comment]
Fixes #449. Also fixes #407 - the event handler there did never work and the injection was borked as well. Follow-up ticket #510 should fix that member groups changes are not persisted across instance restarts!

File: src/ploneintranet/workspace/browser/tiles/sidebar.py [hunk 1]
```diff
@@ -250,6 +190,7 @@ def update_field(field_name):
                 )
 
         if self.request.method == 'POST':
+            self.form_submitted = True
```
Comment (gyst) [inline src/ploneintranet/workspace/browser/tiles/sidebar.py:193]: This fixes the injection

File: src/ploneintranet/workspace/configure.zcml [hunk 2]
```diff
@@ -47,8 +47,7 @@
   <!-- Subscribe to ParticipationPolicyChangedEvent in order to
        update existing users-->
   <subscriber
-     for=".workspacefolder.IWorkspaceFolder
-          .interfaces.IParticipationPolicyChangedEvent"
+     for=".interfaces.IParticipationPolicyChangedEvent"
```
Comment (gyst) [inline src/ploneintranet/workspace/configure.zcml]: This fix is needed to get the subscriber actually called!

**Comment by mattss** [comment]
@gyst - added my fixes for #510 in here. Also reverted your subscriber change. Can't explain why you had to do that I'm afraid, but reverting it makes all tests pass, and UI sidebar still fires the event in my manual testing.

**Comment by mattss** [comment]
I've reviewed the changes made by @gyst in this and I'm happy that the buttons now appear to match the assigned policy.

Someone may want to review my changes before merging.

**Pull request merged**